File: C/financialCalculator.c
#include <stdio.h>

int main(void){
  float income, rent, utilites, groceries, transportation, savings, expenses, total;
  float prent, putilites, pgroceries, ptransportation, psavings, pexpenses;
  printf("Hello and welcome to our budget calculator\n");
  printf("how much do you make each month?:");
  scanf("%f", &income);
  printf("how much is your rent?:");
  scanf("%f", &rent);
  printf("how much is your utilites?:");
  scanf("%f", &utilites);
  printf("how much is your groceries?:");
  scanf("%f", &groceries);
  printf("how much is your transportation?:");
  scanf("%f", &transportation);
  expenses = rent + utilites + groceries + transportation;
  savings = income * .2;
  total = income - expenses - savings;
  printf("Your monthly income is $%.2f\n", income);
  printf("Your monthly expenses is $%.2f\n", expenses);
  printf("Your monthly savings is $%.2f\n", savings);
  printf("you have $%.2f\n", total);
  prent = rent/income * 100;
  putilites = utilites/income * 100;
  pgroceries = groceries/income *100;
  ptransportation = transportation/income * 100;
  pexpenses = expenses/income * 100;
  psavings = savings/income * 100;
  printf("your expenses are %.1f%% of your income\n", pexpenses);
  printf("your rent is %.1f%% of your income\n", prent);
  printf("your groceries are %.1f%% of your income\n", pgroceries);
  printf("your trasportation is %.1f%% of your income\n", ptransportation);
  printf("your savings are %.1f%% of your income\n", psavings);
    return 0;
}

File: Python/financialCalculator.py
print("Welcome to your financial calculator\n")

#Inputs

income = float(input("What's your monthly income?: "))
rent = float(input("What's your monthly rent?: "))
utilities = float(input("What's your monthly utilities?: "))
groceries = float(input("What's your monthly groceries?: "))
transport = float(input("What's your monthly transportation costs?: "))

#Calculatiing Percents

expenses = float(utilities+groceries+transport+rent)
savings = float((income-expenses)/3)
spendings = float(income-expenses-savings)
prent = float(rent/income * 100)
putilities = float(utilities/income * 100)
pgrocieries = float(groceries/income * 100)
ptransportation = float(transport/income * 100)
psavings = float(savings/income * 100)
pexpenses = float(expenses/income * 100)

#Printing Percents

print(f"Your monthly income is ${round(income,3)}\n")
print(f"Your monthly expenses are ${round(expenses,3)}\n")
print(f"Your monthly savings is ${round(savings,3)}\n")
print(f"Your monthly spendings are ${round(spendings,3)}\n")
print(f"Your rent is {int(prent)}% of your monthly income\n")
print(f"Your utilities are {int(putilities)}% of your monthly income\n")
print(f"Your groceries are {int(pgrocieries)}% of your monthly income\n")
print(f"Your transportation is {int(ptransportation)}% of your monthly income\n")
print(f"Your savings are {int(psavings)}% of your monthly income\n")
print(f"Your expenses are {int(pexpenses)}% of your monthly income\n")